src/EventListener/FormSubscriptionListener.php

```diff
@@ -140,7 +140,19 @@ public function onProcessFormData(array $submittedData, array $formData, ?array
         if ($result) {
             $this->logger->log(LogLevel::INFO, sprintf('Successfully subscribed "%s" to Mailchimp list "%s".', $email, $model->listName), ['contao' => new ContaoContext(__METHOD__, TL_GENERAL)]);
         } else {
-            $this->logger->log(LogLevel::INFO, sprintf('Could not subscribe "%s" to Mailchimp list "%s".', $email, $model->listName), ['contao' => new ContaoContext(__METHOD__, TL_ERROR)]);
+            $msg = sprintf('Could not subscribe "%s" to Mailchimp list "%s".', $email, $model->listName);
+
+            if ($lastError = $api->getLastError()) {
+                if ($lastError->detail ?? null) {
```

Comment:
Don't we have to raise the minimum PHP version for this?

Reply:
May be. Which PHP version introduced the null coalescing operator? 😁

Reply:
Hmm, could work :) https://www.php.net/manual/en/migration70.new-features.php looks like PHP 7.0 :)